redaxo/src/addons/metainfo/install.php

```diff
@@ -30,6 +30,7 @@
     ->ensureIndex(new rex_sql_index('name', ['name'], rex_sql_index::UNIQUE))
     ->ensure();
 
+// keep in sync with rex_metainfo_default_type constants
 $data = [
     ['id' => 1, 'label' => 'text', 'dbtype' => 'text', 'dblength' => 0],
```

Comment:
```suggestion
    ['id' => rex_metainfo_default_type::TEXT, 'label' => 'text', 'dbtype' => 'text', 'dblength' => 0],
```

wollen wir hier die constants nicht auch direkt mit durchziehen?

Reply:
getestet.. geht. danke

Reply:
Problem sind Updates: Angenommen eine neue Version führt eine neue Konstante ein, und die nutzen wir dann auch hier in der Datei. Während des Updates ist aber noch das alte Addon geladen, wo die neue Konstante dann fehlt.

Reply:
D.h. Den letzte commit reverten? Oder gibts alternativen ?

Reply:
Entweder reverten. Oder kommentar hier dran, das zukünftige neue konstanten hier nicht direkt verwendet werden können wegen update-routine.